Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -56,14 +59,14 @@ public static FluentLayout WithSameTop(this UIView view, UIView previous)
             return view.Top().EqualTo().TopOf(previous);
         }
 
-        public static FluentLayout WithSameCenterX(this UIView view, UIView previous)
+        public static FluentLayout WithSameCenterX(this UIView view, UIView previous, nfloat? margin = null)
         {
```

Comment:
Is `margin` the right word here though? 

For centers, is this more of an `offset`?

(English language thing only - not a technical note - just a note for someone trying to use/understand the API)